lntest/harness.go

```diff
@@ -36,6 +36,11 @@ const DefaultCSV = 4
 type NetworkHarness struct {
 	netParams *chaincfg.Params
 
+	// name is the "name" of the harness which is currently used to prefix
+	// node log files. Can be set to the testcase name or anything that
+	// gives extra information when looking at logs.
+	name string
```

Comment:
Should we call this `currentTestCase` to make it easier to distinguish from the node's name? Same in the `NodeConfig`, could be `CurrentTestCase` or `LogFilenamePrefix`.

Reply:
Yeah, agree, much better!